src/services/Migrations.ts

```diff
@@ -40,6 +45,151 @@ class Migrations {
       console.log(`collection ${CachedValidator} removed.`);
     });
   };
+
+  private static migrateTo7280 = async () => {
+    await Migrations.wrapMigration('7.28.0', async () => {
+      // we have different index and validator per network, so let's simply drop and rebuild
```

Comment:
is this comment up to date? I do not see you droping anything

Reply:
Removed